networkit/cpp/auxiliary/Random.cpp

```diff
@@ -26,12 +27,15 @@ static std::atomic<bool> seedUseThreadId{false};
 // relaxed.
 // As long as globalSeedGeneration is zero, the getURNG()
 // ignores these two variables.
-static std::atomic<uint64_t> globalSeedGeneration{0}; //< global seed generation, updated on every setSeed-call
+//static std::atomic<uint64_t> globalSeedGeneration{0}; //< global seed generation, updated on every setSeed-call
 
 void setSeed(uint64_t seed, bool useThreadId) {
-    seedValue.store(seed, std::memory_order_relaxed);
-    seedUseThreadId.store(useThreadId, std::memory_order_relaxed);
-    globalSeedGeneration.fetch_add(1, std::memory_order_release) ;
+    Aux::GlobalState::Random::setSeed(seed);
+    //seedValue.store(seed, std::memory_order_relaxed);
+    Aux::GlobalState::Random::setSeedUseThreadId(useThreadId);
+    //seedUseThreadId.store(useThreadId, std::memory_order_relaxed);
+    Aux::GlobalState::Random::incGlobalSeed();
+    //globalSeedGeneration.fetch_add(1, std::memory_order_release);
```

Comment:
Likewise.

Reply:
And everywhere else (do not keep commented out code unless there is a *very* good reason).